src/httprpc.cpp

```diff
@@ -75,6 +79,50 @@ static void JSONErrorReply(HTTPRequest* req, const UniValue& objError, const Uni
     req->WriteReply(nStatus, strReply);
 }
 
+//This function checks username and password against -rpcauth
+//entries from config file.
+static bool multiUserAuthorized(std::string strUserPass)
```

Comment:
This function seems to ignore syntax errors in the file (missing ":" or unexpected number of fields); I think they should be reported.

Reply:
Syntax errors should be checked elsewhere. This function is called when a new JSON-RPC request comes in - we should not be throwing here on syntax errors. Please file a ticket for this.